code/lib/cli/src/initiate.ts

```diff
@@ -323,7 +323,7 @@ async function doInitiate(options: CommandOptions, pkg: PackageJson): Promise<vo
 
     // Add a new line for the clear visibility.
     logger.log();
-    throw new HandledError(`Angular project already installed`);
+    throw new HandledError(`Project already inited with Storybook`);
```

Comment:
A few observations:

1. I think there's a deeper meaning in this error. The "isStorybookInstalled" function checks for a renderer package being installed in package.json, e.g. if people install Storybook for React, it checks for `@storybook/react` existence. But what if you have a monorepo with two React projects, where one project already has Storybook, and you are trying to initialize Storybook on the second project? Seems like this check would throw an error, which would seem quite weird for the user!
2. When looking at the code path, it seems like it would be possible to make this interactive and ask the user whether they want to continue the installation or not, and if the user said yes then the code would just continue. We could do that instead of throwing an error.

What do you think?

Reply:
> When looking at the code path, it seems like it would be possible to make this interactive and ask the user whether they want to continue the installation or not, and if the user said yes then the code would just continue. We could do that instead of throwing an error.

Sounds great!

Reply:
On the other hand, continuing only helps a little because, in a mono repository setup, different framework-specific dependencies might occur in the package.json. Angular might be installed, as well as React. Our auto-detection of the project type would fail gracefully. Instead, in mono repositories, we would have to tell the user to restart the init command and execute it with a project type.

Reply:
WDYT of changing the detection and providing the error message only in scenarios where there are multiple project types then?

Reply:
It doesn't cover all scenarios. It is pretty common to only use one framework in a mono repo setup.

Reply:
I would approach it differently. As soon as we detect a `.storybook` folder, we abort the `initialization`. That's the easiest approach I can imagine. Other approaches are not perfect and require even further spagettify our codebase.

Reply:
Sounds good, should we try that?